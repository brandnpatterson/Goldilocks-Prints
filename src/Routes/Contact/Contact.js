import React from 'react'
import './Contact.sass'

class Contact extends React.Component {
  render () {
    console.log(this.props)
    return (
      <div className='contact'>
        <div className='landing'>
          <h2>Contact me!</h2>
        </div>
        <div className='panel'>
          <h2>Contact</h2>
        </div>
      </div>
    )
  }
}

export default Contact
