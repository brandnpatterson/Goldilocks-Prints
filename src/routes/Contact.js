import React from 'react'

const Contact = () => {
  return (
    <div className='contact'>
      <div className='landing'>
        <h2>Contact Me Today</h2>
      </div>
      <div className='panel'>
        <h2>Contact</h2>
      </div>
    </div>
  )
}

export default Contact
