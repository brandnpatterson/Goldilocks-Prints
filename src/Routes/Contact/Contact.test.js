import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import Contact from './Contact'

test('It should render the Contact component as expected', () => {
  const component = shallow(<Contact />)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})
