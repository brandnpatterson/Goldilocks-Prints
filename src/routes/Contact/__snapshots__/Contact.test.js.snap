// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should render the Contact component as expected 1`] = `
<div
  className="contact"
>
  <div
    className="landing"
  >
    <h2>
      Contact me!
    </h2>
  </div>
  <div
    className="panel"
  >
    <h2>
      Contact
    </h2>
  </div>
</div>
`;
